*** file_ops.tcl.bk	Fri Sep 20 18:24:48 1996
--- file_ops.tcl	Fri Sep 20 18:24:48 1996
***************
*** 1472,1477 ****
--- 1472,1507 ----
      set dsk_delete(all) $tkdesk(all_files)
      set dsk_delete(really) $tkdesk(really_delete)
  
+ #
+ # modifications by jdblair
+ # support a new config: "action_on_delete"
+ #
+ 
+ 	if [info exists tkdesk(action_on_delete)] {
+ 		set newd [string trimright $files /]
+ 		set dmatch 0
+ 		foreach d $tkdesk(action_on_delete) {
+ 		set dp [lindex $d 0]  ;# list of patterns
+ 		set da [lindex $d 1]  ;# Tcl script to execute
+ 		foreach p $dp {
+ 			set p [string trimright [cb_tilde $p expand] /]
+ 			if [string match $p $newd] {
+ 				set err [catch {eval [_expand_pc $da]} errmsg]
+ 				if $err {
+ 					dsk_errbell
+ 					cb_error $errmsg
+ 				}
+ 				set dmatch 1
+ 				break
+ 			}
+ 		}
+ 		if $dmatch break }
+ 	} else {
+ 	
+ #
+ # end modifications by jdblair
+ #
+ 
      if {[string first "$tkdesk(configdir)/.trash" [lindex $files 0]] > -1} {
  	set dsk_delete(really) 1
      }
***************
*** 1555,1560 ****
--- 1585,1594 ----
      wm protocol $t WM_DELETE_WINDOW {.dsk_delete.bCancel invoke}
  
      dsk_place_window $t dsk_delete 470x98
+ 
+ # jdblair mod:
+ # this trailing close brace ends the original delete functions
+     }
  }
  
  proc dsk_delete_action {cmd} {
