*** /usr/tmp/TclTk/blt2.4j/src/bltDnd.c	Tue Sep 21 20:15:34 1999
--- bltDnd.c	Sun Nov  7 19:38:52 1999
***************
*** 1614,1619 ****
--- 1614,1623 ----
      }
      targetInfo = srcPtr->windowPtr->targetInfo;
  
+     /* Let TkDesk know we found a valid drop target */
+     Tcl_SetVar2(srcPtr->interp, "tkdesk", "drop_on_item", "1",
+ 		TCL_GLOBAL_ONLY);
+ 
      Tcl_DStringInit(&dString);
      Blt_DStringAppendElements(&dString, "send", targetInfo[INTERP_NAME],
  	dragDropCmd, "location", (char *)NULL);
