*** /usr/tmp/TclTk/blt2.4j/src/bltInt.h	Tue Sep 21 20:15:40 1999
--- bltInt.h	Thu Oct 28 22:19:31 1999
***************
*** 41,57 ****
--- 41,60 ----
  #define TCL_VERSION_NUMBER _VERSION(TCL_MAJOR_VERSION, TCL_MINOR_VERSION, TCL_RELEASE_SERIAL)
  #define TK_VERSION_NUMBER _VERSION(TK_MAJOR_VERSION, TK_MINOR_VERSION, TK_RELEASE_SERIAL)
  
+ #include "config.h"
  #include "bltTkInt.h"
  
  #include <stdio.h>
  #include <assert.h>
  #include <math.h>
  
+ /*
  #ifdef WIN32
  #include "bltWinConfig.h"
  #else
  #include "bltConfig.h"
  #endif
+ */
  
  #include "blt.h"
  
