*** /usr/tmp/TclTk/blt2.4j/src/Makefile.in	Tue Sep 21 20:15:29 1999
--- Makefile.in	Thu Oct 28 22:28:01 1999
***************
*** 1,158 ****
! # ------------------------------------------------------------------------
! # 	Makefile for static version of BLT library
! # ------------------------------------------------------------------------
! 
! # ------------------------------------------------------------------------
! # 	C Compiler options 
! # ------------------------------------------------------------------------
  
! BLT_LIBRARY  = @BLT_LIBRARY@
! CC           = @CC@
! CFLAGS       = @CFLAGS@ 
! DEFINES      = @DEFINES@
! DEF_BLTINIT  = -DBLT_LIBRARY=\"$(BLT_LIBRARY)\"
! EXTRA_CFLAGS = @GCCFLAGS@
! LDFLAGS      = @LDFLAGS@
! version      = @BLT_MAJOR_VERSION@@BLT_MINOR_VERSION@
! 
! # ------------------------------------------------------------------------
! # 	Source and targer installation directories 
! # ------------------------------------------------------------------------
! 
! bindir       = @bindir@
! exec_prefix  = @exec_prefix@
! includedir   = @includedir@
! libdir	     = @libdir@
! prefix	     = @prefix@
! srcdir	     = @srcdir@
! 
! instdirs  = $(prefix) $(exec_prefix) $(bindir) $(libdir) $(includedir)
! 
! # ------------------------------------------------------------------------
! # 	Directories containing Tcl and Tk include files and libraries
! # ------------------------------------------------------------------------
! 
! INCLUDES	= -I. -I$(srcdir) @INCLUDES@
! 
! # ------------------------------------------------------------------------
! #       Libraries directives for Tcl, Tk, X11, and BLT
! # ------------------------------------------------------------------------
! 
! LIBRARIES    = $(libname).a @LIBS@
! 
! 
! # ------------------------------------------------------------------------
! # 	You don't need to edit anything beyond this point
! # ------------------------------------------------------------------------
! 
! #N_OBJS		= bltTed.o
! V3_OBJS		= bltTri.o bltGrMt.o 
! 
! TK_OBJS		= tkButton.o tkFrame.o tkScrollbar.o 
! 
! OBJS =		bltGraph.o bltGrAxis.o bltGrBar.o bltGrElem.o \
! 		bltGrGrid.o bltGrHairs.o bltGrLegd.o bltGrLine.o \
! 		bltGrMarker.o bltGrMisc.o bltGrPen.o bltGrPs.o \
! 	       	bltBeep.o bltBgexec.o bltBitmap.o bltBusy.o \
! 		bltContainer.o bltCutbuffer.o bltDebug.o \
!                 bltDnd.o bltUnixDnd.o \
! 		bltHierbox.o bltHtext.o bltSpline.o bltTable.o bltTabset.o \
! 		bltVector.o bltWatch.o bltWinop.o \
! 		bltCanvEps.o bltInit.o bltBind.o bltColor.o \
! 		bltConfig.o bltImage.o bltList.o bltChain.o \
! 		bltNsUtil.o bltParse.o bltPs.o bltText.o bltTile.o \
! 		bltUnixPipe.o bltUtil.o bltWindow.o \
! 		$(TK_OBJS) $(N_OBJS) 
! 
! # GNU Make-specific macro
! SRCS 		= $(patsubst %.o,$(srcdir)/%.c,$(OBJS)) 
! 
! demo		= bltwish
! headers		= blt.h
! libname  	= libBLT
! 
! CC_SWITCHES	= $(EXTRA_CFLAGS) $(CFLAGS) $(DEFINES) $(INCLUDES)
! INSTALL  	= @INSTALL@
! INSTALL_DATA  	= @INSTALL_DATA@
  RANLIB   	= @RANLIB@
! SHELL    	= /bin/sh
  AR       	= ar rc
  RM       	= rm -f
- LINT		= lint
- LINTFLAGS	= -axhbns
- LN_S		= @LN_S@
- VPATH		= $(srcdir)
  
! all: build-lib build-demo @SHLIB_TARGET@
  
! build-lib: $(libname).a
  
! build-shared:
! 	(cd shared; $(MAKE) CFLAGS="$(CFLAGS)" all) 
  
! build-demo: $(demo)
  
! $(demo): $(libname).a
! 	$(RM) $(demo)
! 	$(CC) $(CC_SWITCHES) $(LDFLAGS) \
! 		$(srcdir)/bltUnixMain.c -o $(demo) $(LIBRARIES) 
  
! $(libname).a:  $(OBJS)
  	$(RM) $@
  	$(AR) $@ $(OBJS)
  	$(RANLIB) $@
  
- install: install-dirs install-lib install-demo install-headers
- 
- install-demo: $(demo)
- 	$(INSTALL) -m 0755 bltwish $(bindir)/bltwish$(version)
- 	(cd $(bindir); $(RM) bltwish ; $(LN_S) bltwish$(version) bltwish)
- 
- install-lib: $(libname).a
- 	$(INSTALL_DATA) $(libname).a $(libdir)/$(libname)$(version).a
- 	(cd $(libdir); $(RM) $(libname).a ; \
- 		$(LN_S) $(libname)$(version).a $(libname).a)
- 	$(RANLIB) $(libdir)/$(libname)$(version).a
- 	(cd shared; $(MAKE) install) 
- 
- install-dirs:
- 	@for i in $(instdirs) ; do \
- 	    if test ! -d $$i ; then \
- 		echo " mkdir $$i" ; \
- 		mkdir $$i ; \
- 	    fi ; \
-         done
- 
- install-headers:
- 	@for i in $(headers) ; do \
- 	    echo "installing $(includedir)/$$i..." ; \
- 	    $(INSTALL_DATA) -m 0444 $(srcdir)/$$i $(includedir) ; \
- 	done
- 
- lint:
- 	$(LINT) $(LINTFLAGS) $(DEFINES) $(INCLUDES) $(SRCS)
- 
- clean:
- 	$(RM) $(OBJS) $(DEMO_OBJS) $(libname).a $(demo)* *pure* .pure*
- 	(cd shared; $(MAKE) clean) 
- 
- distclean: clean
- 	$(RM) $(srcdir)/*.bak $(srcdir)/*\~ $(srcdir)/"#"*  Makefile
- 	$(RM) bltConfig.h Makefile TAGS
- 
- 
  bltInit.o: bltInit.c
! 	$(CC) -c $(CC_SWITCHES) $(DEF_BLTINIT) $<
  
! .c.o:
! 	$(CC) -c $(CC_SWITCHES) $<
! 
! PUREFLAGS=
! pure: $(libname).a
! 	$(PURIFYHOME)/purify $(PUREFLAGS) $(CC) $(CC_SWITCHES) \
! 		$(srcdir)/bltUnixMain.c -o bltwish $(LIBRARIES)
! 
! QUANTIFYFLAGS=
! quant: $(libname).a
! 	$(QUANTIFYHOME)/quantify $(QUANTIFYFLAGS) $(CC) $(CC_SWITCHES) \
! 		$(srcdir)/bltUnixMain.c -o bltwish $(LIBRARIES)
--- 1,55 ----
! # This is the Makefile for the BLT subset needed by TkDesk.
! # It is called from the main Makefile in .., which passes the CFLAGS.
  
! #----------------------------------------------------------------
! # Things you can change to personalize the Makefile for your own
! # site (you can make these changes in either Makefile.in or
! # Makefile, but changes to Makefile will get lost if you re-run
! # the configuration script).
! #----------------------------------------------------------------
! 
! # Location of Tcl header files:
! TCL_INCLUDE_DIR =	@TCL_INCLUDE_PATH@
! 
! # Location of Tk header files:
! TK_INCLUDE_DIR =	@TK_INCLUDE_PATH@
! 
! # Location of X11 header files:
! X_INCLUDE_FLAG =	@TK_XINCLUDES@
! 
! # Configuration compiler flags:
! AC_FLAGS =		@DEFS@
! 
! # Miscellaneous settings:
  RANLIB =		@RANLIB@
! CC =			@CC@
  AR =			ar rc
  RM =			rm -f
  
! #----------------------------------------------------------------
! # The information below should be usable as is.  The configure
! # script won't modify it and you shouldn't need to modify it
! # either.
! #----------------------------------------------------------------
  
! CFLAGS =	${CC_OPTS} ${AC_FLAGS} -I.. -I. -I${TCL_INCLUDE_DIR} -I${TK_INCLUDE_DIR} ${X_INCLUDE_FLAG}
! BLT_LIBRARY =	$(LIB_DIR)
  
! libname	=	libBLT.a
  
! OBJS = 		bltBgexec.o bltBusy.o bltUnixPipe.o \
! 		bltText.o bltDnd.o bltInit.o bltList.o bltUtil.o \
! 		bltConfig.o bltWindow.o bltChain.o bltUnixDnd.o
  
! lib: $(libname)
  
! $(libname):  $(OBJS)
  	$(RM) $@
  	$(AR) $@ $(OBJS)
  	$(RANLIB) $@
  
  bltInit.o: bltInit.c
! 	$(CC) -c $(CFLAGS) -DBLT_LIBRARY=\"$(BLT_LIBRARY)\" $<
  
! clean:
! 	$(RM) $(OBJS) $(libname) *\~ "#"*
